dd-sdk-android/src/test/kotlin/com/datadog/android/webview/WebViewEventConsumerTest.kt

```diff
@@ -140,7 +140,7 @@ internal class WebViewEventConsumerTest {
         testedWebViewEventConsumer.consume(fakeWebEvent.toString())
 
         // Then
-        verify(mockSdkLogHandler).handleLog(
+        verify(mockedSdkLogHandler).handleLog(
```

Comment:
This renaming doesn't follow our [guidelines](https://github.com/DataDog/dd-sdk-android/blob/master/CONTRIBUTING.md#fields)

Reply:
Yes but if I follow the guidelines it will become : `mockSdkLogHandler(mockSdkLogHandler)`. I do not mind this method/name clash but I had @0xnm commenting above on this part.

Reply:
I was thinking this clash leads to the compilation failure from reading the code, but apparently compiler is fine with that. So I'm ok with that as well.